examples/algorithms/lra.cpp

```diff
@@ -38,11 +38,9 @@ char const* const lra_code = R"(
 
 int main(int argc, char* argv[])
 {
-    Eigen::VectorXd v1(8, 1);
-    v1 << 17.99, 20.57, 19.69, 11.42, 20.29, 12.45, 18.25, 13.71;
+    blaze::DynamicVector<double, blaze::rowVector> v1({ 17.99, 20.57, 19.69, 11.42, 20.29, 12.45, 18.25, 13.71});
```

Comment:
Please try to avoid lines longer than 80-90 characters. I will add inspect to the CI builds soon - this will flag these things.